repository/index.php

```diff
@@ -245,6 +245,59 @@ function getLastRevision($version) {
 <li><a href="https://scan.coverity.com/projects/llvm">Coverity reports</a></li>
 </ul>
 </div>
+
+<div class="rel_section">
+Building the latest nightly snapshot
+</div>
+
+<div class="rel_boxtext">
+<p>The latest nightly snapshot can be built on your own machine with the following steps. First, ensure you add to your apt.sources the <a href="http://llvm.org/apt">nightly repositories for your distribution</a>.</p>
+<p>Use apt-get to retrieve the sources of the llvm-toolchain-snapshot package,</p>
+<p class="www_code"> $ mkdir build/ &amp;&amp; cd build/ <br />
+ $ apt-get source llvm-toolchain-snapshot </p>
+<p>This should download all the original snapshot tarballs, and create a directory named llvm-toolchain-snapshot-3.9~svn270412. Depending on the last update of the jenkins nightly builder, the snapshot version number and svn release will vary.</p>
+<p>Then install the build dependencies,</p>
+<p class="www_code"> $ sudo apt-get build-dep llvm-toolchain-snapshot </p>
```

Comment:
That doesn't always work when you are using old ubuntu because some gcc versions are not available, users must also add https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/ppa to get the packages